src/main/java/app/WebConfig.java

```diff
@@ -0,0 +1,17 @@
+package app;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class WebConfig implements WebMvcConfigurer {
+  @Override
+  public void addResourceHandlers(ResourceHandlerRegistry registry) {
+    registry.addResourceHandler("/webjars/**", "/images/**")
+        .addResourceLocations("classpath:/META-INF/resources/webjars/", "classpath:/static/images/")
+        .resourceChain(false);
```

Comment:
I don't think this setting is necessary.

Reply:
For render images in tree items. Image resource was uploaded from HTML file:

>const img = PIXI.Texture.from('images/file_black.png', {crossOrigin: 'anonymous'});
> const tree = new wcardinal.ui.DTree({
> 				...
> 				value: [
> 					{
> 					text: "item0",
> 					image: img,
> 					children: [
> ...

I need WebConfig.java class to access image resource in Spring,

Reply:
As far as I know, we can access to resources under the `static` directory thanks to the Spring Boot's auto configuration feature by default.

Reply:
Thank you for your comment.
I will remove this setting and move image resources to default folder